Formula/subversion.rb

```diff
@@ -126,6 +116,9 @@ def install
       --enable-optimize
       --with-zlib=/usr
       --with-sqlite=#{Formula["sqlite"].opt_prefix}
+      --with-apr=#{Formula["apr"].opt_prefix}
+      --with-apr-util=#{Formula["apr-util"].opt_prefix}
+      --with-apxs=no
```

Comment:
Yeah, we put it in a while back. Pretty sure someone gave me a detailed explanation as to why it would be needed at the time, but by now it has well & truly escaped my brain.

To be honest though I've never really had cause to question or look into it much because it has always seemed to work & nobody's gone _"Hey why doesn't xyz configuration handle xyz use case"_ etc.